CMakeLists.txt

```diff
@@ -22,29 +22,21 @@ http://cgit.freedesktop.org/~mareko/libtxc_dxtn/" ON)
 
 option(BUILD_DOCS "Build doxygen documentation" OFF)
 
-if(USE_NVDXT AND USE_LIBTXC_DXTN)
-	message(FATAL_ERROR "Cannot use both options USE_NVDXT and USE_LIBTXC_DXTN at once.")
+if(USE_NVDXT)
+	message(FATAL_ERROR "USE_NVDXT is not longer supported.")
 endif()
 
 # ---- dependencies -----------------------------------------------------------
-if(USE_LIBTXC_DXTN)
-	find_path(GL_INCLUDE_DIR GL/gl.h)
-	find_library(LIBTXC_DXTN_LIBRARY txc_dxtn)
+find_path(GL_INCLUDE_DIR GL/gl.h)
 
-	if(NOT(GL_INCLUDE_DIR))
-		message(FATAL_ERROR "GL/gl.h not found!
+if(NOT(GL_INCLUDE_DIR))
+    message(FATAL_ERROR "GL/gl.h not found!
```

Comment:
This is probably the problem with the builds, this is not defined so it errors. Is this needed or can it be removed now?